source/agora/crypto/Types.d

```diff
@@ -42,3 +42,10 @@ public enum PrintMode
     /// Print original value
     Clear,
 }
+
+/// Whether integers are serialized in variable-length form
+public enum CompactMode : bool
+{
+    No,
+    Yes
```

Comment:
```suggestion
    Yes,
```

Reply:
As `yes` or `no` enum it is unlikely to ever get more options so I'll leave this time.

Reply:
https://thedailywtf.com/articles/What_Is_Truth_0x3f_